tensorflow_datasets/scripts/random_image_generator.py

```diff
@@ -0,0 +1,102 @@
+import os
+from zipfile import ZipFile
+from PIL import Image
+import numpy as np
+import tarfile
+import zipfile
+import shutil
+import gzip
+
+
+# images_path variable to store the path of images for later compression
+images_path = []
+
+
+def image_process(root):
+    """ Generate random images and remove noise of the
+        images to compress better.
+
+          Args:
+           root: path of the images to get processed
+
+    """
+    images_path.append(root)
+    image = np.array(Image.open(root))
+    image = np.ones_like(image) * np.random.randint(np.amax(image))
```

Comment:
As explained previously: For the random generation, rather than pure randomness, we should make it deterministic such as running the script twice produce the same image. One way is to use the hash of the relative path % 255. As the relative path is constant across run, the images will be too.

Reply:
Done